Bai-tap-Function-2/Bai3.py

```diff
@@ -0,0 +1,17 @@
+def print_rec(m, n):
+  """
+  In ra hình chữ nhật có chiều dài m và chiều rộng n.
+
+  Args:
+    m: Chiều dài của hình chữ nhật.
+    n: Chiều rộng của hình chữ nhật.
+  """
+  if m <= n:
+        print('Fail')
+  else:   
+    for i in range(n):
+        for j in range(m):
+         print('*', end='')
+        print() 
```

Comment:
@ngochan21098 
It's so good. But it's another way below:

```suggestion
        print('*' * m)
```

Reply:
@ddhienpy I have understood your comment and update code here 
https://github.com/ngochan21098/learning-python-basic/pull/1/commits/258d14d70a1a8b45cb0ddeb15130b4df11cf3601